genesis.tf

```diff
@@ -21,7 +21,8 @@ resource "digitalocean_droplet" "testnet_genesis" {
   provisioner "remote-exec" {
     inline = [
       "chmod +x /tmp/init-node.sh",
-      "/tmp/init-node.sh \"${var.node_url}\" true \"${self.ipv4_address}\" \"${digitalocean_droplet.testnet_genesis.ipv4_address}\" \"${var.port}\" \"${var.remote_log_level}\"",
+      "/tmp/init-node.sh \"${var.node_url}\" true \"${self.ipv4_address}\" \"${digitalocean_droplet.testnet_genesis.ipv4_address}\" \"${var.port}\" \"${var.remote_log_level}\" \"${name}\" \"${var.otlp_collector_endpoint}\"",
```

Comment:
How is `${name}` being set here? These need to be Terraform variables.

Reply:
The `name` is set in L3 as `name = "${terraform.workspace}-safe-node-1"`. I assumed it can be called directly after referring to L49 of `node.tf` (`echo "node-${count.index + 2}`, where `count` is defined inside the same scope)

Reply:
Right. Did you test that? `count` is a special variable, I'm not sure that `name` would also behave like that, but cool if it does!

Reply:
Update: We setup the terraform env, and it turns out that we cannot use the `name` var inside here. 
I've made the changes to the PR..